.github/config.yml

```diff
@@ -0,0 +1,35 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats 🎉 for opening your very first issue in this project.AnitaB.org is an inclusive community, commited to creating a safe and positive environment.🌸 Please adhere to our [Code of Conduct](https://github.com/anitab-org/open-source-programs-web/blob/develop/CODE_OF_CONDUCT.md).🙌
```

Comment:
@Amulya-coder  just noticed this could you please add a space after `project. AnitaB.org`